package.json

```diff
@@ -157,13 +157,13 @@
 		"clean:packages": "rimraf ./packages/*/build ./packages/*/build-module ./packages/*/build-style ./packages/*/node_modules",
 		"prebuild:packages": "npm run clean:packages && lerna run build",
 		"build:packages": "node ./bin/packages/build.js",
-		"build": "npm run build:packages && wp-scripts build",
+		"build": "wp-scripts build",
```

Comment:
At the moment `lerna run build` is needed only for one package which isn't part of the application so it's fine as is.